chartjs-adapter-luxon 1.2.0 is incompatible with chart.js 4.x

I'm seeing the following error when I attempt to update to chart.js 4.x, released ~5 hours ago:

```
D:\GitHub\chart>npm install chart.js@latest chartjs-adapter-luxon@latest
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: stl-status-chart@1.0.0
npm ERR! Found: chart.js@4.0.1
npm ERR! node_modules/chart.js
npm ERR!   chart.js@"4.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer chart.js@"^3.0.0" from chartjs-adapter-luxon@1.2.0
npm ERR! node_modules/chartjs-adapter-luxon
npm ERR!   chartjs-adapter-luxon@"1.2.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

This is similar to #61 which happened when luxon updated its major version.

--- a/karma.conf.js
+++ b/karma.conf.js
@@ -55,7 +55,7 @@ module.exports = function(karma) {
 
     files: [
       {pattern: 'node_modules/luxon/build/global/luxon.js'},
-      {pattern: 'node_modules/chart.js/dist/chart.js'},
+      {pattern: 'node_modules/chart.js/dist/chart.umd.js'},
       {pattern: 'src/index.js', watched: false},
       {pattern: 'test/index.js'},
       {pattern: 'test/specs/**/**.js'}

--- a/package.json
+++ b/package.json
@@ -52,7 +52,7 @@
     "yargs": "^17.0.1"
   },
   "peerDependencies": {
-    "chart.js": "^3.0.0",
+    "chart.js": ">=3.0.0",
     "luxon": ">=1.0.0"
   }
 }

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
   },
   "devDependencies": {
     "@rollup/plugin-node-resolve": "^13.0.0",
-    "chart.js": "^3.9.0",
+    "chart.js": "^4.0.0",
     "chartjs-test-utils": "^0.3.0",
     "concurrently": "^6.1.0",
     "cross-env": "^7.0.3",
@@ -45,7 +45,7 @@
     "karma-jasmine": "^4.0.1",
     "karma-jasmine-html-reporter": "^1.5.4",
     "karma-rollup-preprocessor": "^7.0.7",
-    "luxon": "^3.0.0",
+    "luxon": "^3.1.0",
     "rollup": "^2.38.4",
     "rollup-plugin-istanbul": "^3.0.0",
     "rollup-plugin-terser": "^7.0.2",

